model/scikit/dffml_model_scikit/scikit_model_scorer.py

```diff
@@ -0,0 +1,57 @@
+from dffml.base import config
+from dffml.util.entrypoint import entrypoint
+from dffml.source.source import SourcesContext
+from dffml.model import ModelContext, ModelNotTrained
+from dffml.accuracy import (
+    AccuracyScorer,
+    AccuracyContext,
+)
+
+
+class ScorerWillNotWork(Exception):
+    pass
+
+
+@config
+class SklearnModelAccuracyConfig:
+    pass
+
+
+class SklearnModelAccuracyContext(AccuracyContext):
+    """
+    Default scorer for Sklearn Models.
+    """
+
+    async def score(self, mctx: ModelContext, sctx: SourcesContext):
+        if not mctx._filepath.is_file():
+            raise ModelNotTrained("Train model before assessing for accuracy.")
+
+        if mctx.clf._estimator_type not in ("classifier", "regressor"):
+            raise ScorerWillNotWork(
+                "SklearnModelAccuracy will not work with Clustering Models"
```

Comment:
Based on if statement let's make this:

```suggestion
                "SklearnModelAccuracy only works with Classifiers and Regressors"
```

Just in case there are more types in the future. Keeps the logic and the message the same